### ISSUE TITLE
[Question] How to create a progress bar for undetermined?

### ISSUE THREAD

**Issue by eestein** [creativetimofficial/ct-material-dashboard-pro-angular#334]
### Version
2.3.0
### Reproduction link
http://not-required.com
### Operating System
macOS
### Device
Doesn't matter
### Browser & Version
Any
### Steps to reproduce
Not necessary
### What is expected?
Progress bar running continuosly
### What is actually happening?
Nothing

---

**Comment by eestein** [comment]
Any news on this?

**Comment by sylvaingirardbe** [comment]
Yeah, don't expect too much support from them... Anyway, you can just use `<mat-progress-bar mode="indeterminate"></mat-progress-bar>`.

**Comment by eestein** [comment]
That's said, the product *wasn't free*!
Anyway, I'm not using material, so it won't work for me, I was looking for a OOT solution.

Thanks for your comment

**Comment by sylvaingirardbe** [comment]
Well, since the template is a material design template, I'd say the material solution is pretty much OOTB :). Sorry I can't be of any more help.

**Comment by chelaruc** [comment]
@eestein you can use the nb-bootstrap progress bar: https://ng-bootstrap.github.io/#/components/progressbar/examples

**Comment by eestein** [comment]
@chelaruc is there no CSS way of doing so? I'd like to just add a class to a div and have it display the "always loading" behavior.

**Comment by chelaruc** [comment]
@eestein this is a feature request and usually, we implement it when we have more than 5-10 requests. For now, I'll let this issue open.

**Comment by eestein** [comment]
Ok, your process is reasonable, but IMO that should've been a day 0 feature.

**Comment by chelaruc** [comment]
@eestein check the bootstrap progress bar: https://getbootstrap.com/docs/4.3/components/progress/#animated-stripes
